pkgs/by-name/ka/kana/package.nix

```
@@ -0,0 +1,62 @@
+{ lib
+, stdenv
+, fetchFromGitLab
+, rustPlatform
+, meson
+, ninja
+, pkg-config
+, rustc
+, cargo
+, wrapGAppsHook4
+, desktop-file-utils
+, libadwaita
+, gst_all_1
+}:
+
+stdenv.mkDerivation rec {
```

What about finalAttrs?

```suggestion
stdenv.mkDerivation (finalAttrs: {
```

---

Not a fan of this... Do we currently have specifications that require us to write this way? Do we also need to migrate existing packages to this way of writing?

---

No. Do as you wish.